lib/unsanitized.js

```diff
@@ -0,0 +1,83 @@
+// names of escaping functions that we acknowledge
+const VALID_ESCAPERS = ["Sanitizer.escapeHTML", "escapeHTML"];
+const VALID_UNWRAPPERS = ["Sanitizer.unwrapSafeHTML", "unwrapSafeHTML"];
+
+class Unsanitized {
```

Comment:
Maybe one of these perhaps:
- ruleHelper.js and `ruleHelper = new RuleHelper(context)`?
- utils.js and `utilHelper = new UtilHelper(context)`?
- utils.js and `utilContext = new UtilHelper(context)`?

or

```
const ruleInstance = new RuleHandler(context);
return {
  AssignmentExpression: ruleInstance.AssignmentExpression
}
```
Where `RuleHandler` has all the rules into one big file.